packages/opencensus-web-instrumentation-zone/src/zone-types.ts

```diff
@@ -51,4 +58,7 @@ export declare interface WindowWithOcwGlobals extends Window {
    * specified sample rate. If not specified, a default sampling rate is used.
    */
   ocSampleRate?: number;
+
+  //
```

Comment:
Could you add some documentation here to explain that this controls the regex of what origins will get the trace context header sent to them? Maybe also rename it to be a bit clearer there, e.g. called `ocTraceHeaderHostRegex` or similar?